.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,26 @@
+name: golangci-lint
+on:
+  push:
+    paths:
+      - '**.go'
```

Comment:
```suggestion
```
I think it's not necessary on every push, maybe the PR has not been open yet. With the `pull_request` trigger only it will be just enought

Reply:
You're welcome. I kinda found your issue really fast when using the github search tool for the 'hacktoberfest' label (it was literally on top of the list).

I agree with your ideas. I'll update the PR right away.